### PR TITLE
dataset: implement naive shuffling

### PR DESCRIPTION
**Pull request by indutny**
cc @ry

### CODE REVIEW

**Comment by indutny** [comment]
@ry perhaps a bit later :wink:

File: src/dataset.ts [hunk 1]
```diff
@@ -158,7 +158,7 @@ class SliceDataset extends Dataset {
 
 // TODO
 function stack(tensors: Tensor[], axis = 0): Tensor {
-  throw Error("not implemented");
+  return tensors[0].concat(axis, tensors[1], ...tensors.slice(2));
```
Comment (ry) [inline src/dataset.ts]: Leave as unimplemented - doesn't seem like you're using it

Comment (indutny) [reply src/dataset.ts]: I'm not, but `BatchDataset` indirectly does.

Comment (ry) [reply src/dataset.ts]: also it's incorrect : ) it needs expandDim. and it should be in tensor.ts

**Comment by ry** [comment]
Will land in #398 - Thanks Fedor!!

**Pull request closed without merging**